packages/react/src/reconciler.ts

```diff
@@ -133,6 +139,12 @@ const reconciler = reactReconciler<
   scheduleDeferredCallback() {},
   cancelDeferredCallback() {},
 
+  // @ts-ignore -- @types/react-reconciler is several versions out of date
```

Comment:
Still needed given the change to the version constraint?

Reply:
yeah, because @types/react-reconciler is very far behind -- when/if it ever gets updated (to at least 0.20.x I think), this won't be needed